src/coffee/card.coffee

```diff
@@ -296,6 +298,10 @@ class Card
 
       outEl.textContent = outVal
 
+  getCardType: ->
+    ccType = Payment.fns.cardType(@$numberInput[0].value)
+    if ccType then Payment.fns.cardType(@$numberInput[0].value) else 'unknown'
```

Comment:
Why not `if ccType then ccType else 'unknown'` ??? Why call the function again?

Reply:
Yeah. My bad.